### PR TITLE
[all hosts] (Manifest, Requirement sets) Moving manifest and req set content to this repo

### PR DESCRIPTION
**Pull request by AlexJerabek**
[Staging site](https://review.docs.microsoft.com/en-us/javascript/api/requirement-sets?view=excel-js-preview&branch=AlexJ-Manifest)

This PR moves the manifest and requirement set information to the reference repo. This is thematically fitting and should help with discovery.

There isn't really net-new content in this PR, but I'm happy to add some to overview pages, if it seems helpful.

### CODE REVIEW

File: docs/toc.yml [hunk 1]
```diff
@@ -66,6 +67,8 @@ items:
     - name: Overview
       href: develop/develop-overview.md
       displayName: Excel, Word, OneNote, Project, PowerPoint, Outlook
+    - name: Api reference
```
Comment (samantharamon) [inline docs/toc.yml:70]:
```suggestion
    - name: API reference
```

File: docs/testing/troubleshoot-development-errors.md [hunk 2]
```diff
@@ -60,10 +60,10 @@ The following are some of the causes of this error. If you discover additional c
...
-- The add-in's manifest has a `resid` that is not defined anywhere in the [Resources](../reference/manifest/resources.md) section of the manifest, or there is a mismatch in the spelling of the `resid` between where it is used and where it is defined in the `<Resources>` section.
+- The value of the [ID](/javascript/api/manifest/id.md) element in the manifest has been changed directly in the deployed copy. If for any reason, you want to change this ID, first remove the add-in from the Office host, then replace the original manifest with the changed manifest. You many need to clear the Office cache to remove all traces of the original. See the [Clear the Office cache](clear-cache.md) article for instructions on clearing the cache for your operating system.
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/troubleshoot-development-errors.md:63]:
```suggestion
- The value of the [ID](/javascript/api/manifest/id) element in the manifest has been changed directly in the deployed copy. If for any reason, you want to change this ID, first remove the add-in from the Office host, then replace the original manifest with the changed manifest. You many need to clear the Office cache to remove all traces of the original. See the [Clear the Office cache](clear-cache.md) article for instructions on clearing the cache for your operating system.
```

File: docs/testing/troubleshoot-development-errors.md [hunk 3]
```diff
@@ -60,10 +60,10 @@ The following are some of the causes of this error. If you discover additional c
...
+- The add-in's manifest has a `resid` that is not defined anywhere in the [Resources](/javascript/api/manifest/resources.md) section of the manifest, or there is a mismatch in the spelling of the `resid` between where it is used and where it is defined in the `<Resources>` section.
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/troubleshoot-development-errors.md:64]:
```suggestion
- The add-in's manifest has a `resid` that is not defined anywhere in the [Resources](/javascript/api/manifest/resources) section of the manifest, or there is a mismatch in the spelling of the `resid` between where it is used and where it is defined in the `<Resources>` section.
```

File: docs/testing/troubleshoot-development-errors.md [hunk 4]
```diff
@@ -60,10 +60,10 @@ The following are some of the causes of this error. If you discover additional c
...
 - There is a `resid` attribute somewhere in the manifest with more than 32 characters. A `resid` attribute, and the `id` attribute of the corresponding resource in the `<Resources>` section, cannot be more than 32 characters.
-- The add-in has a custom Add-in Command but you are trying to run it on a platform that doesn't support them. For more information, see [Add-in commands requirement sets](../reference/requirement-sets/add-in-commands-requirement-sets.md).
+- The add-in has a custom Add-in Command but you are trying to run it on a platform that doesn't support them. For more information, see [Add-in commands requirement sets](/javascript/api/requirement-sets/add-in-commands-requirement-sets.md).
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/troubleshoot-development-errors.md:66]:
```suggestion
- The add-in has a custom Add-in Command but you are trying to run it on a platform that doesn't support them. For more information, see [Add-in commands requirement sets](/javascript/api/requirement-sets/add-in-commands-requirement-sets).
```

File: docs/word/create-better-add-ins-for-word-with-office-open-xml.md [hunk 5]
```diff
@@ -12,7 +12,7 @@ ms.localizationpriority: medium
...
 
-So what are your options when you need to add rich content to a document, such as images, formatted tables, charts, or even just formatted text? Start with the APIs available through the [WordApi requirement sets](../reference/requirement-sets/word-api-requirement-sets.md) to see if they provide what you need. For an example, see the [Insert formatted text](https://github.com/OfficeDev/office-js-snippets/blob/prod/samples/word/25-paragraph/insert-formatted-text.yaml) code snippet in [Script lab](https://appsource.microsoft.com/product/office/wa104380862) on Word.
+So what are your options when you need to add rich content to a document, such as images, formatted tables, charts, or even just formatted text? Start with the APIs available through the [WordApi requirement sets](/javascript/api/requirement-sets/word-api-requirement-sets.md) to see if they provide what you need. For an example, see the [Insert formatted text](https://github.com/OfficeDev/office-js-snippets/blob/prod/samples/word/25-paragraph/insert-formatted-text.yaml) code snippet in [Script lab](https://appsource.microsoft.com/product/office/wa104380862) on Word.
```
Comment (ElizabethSamuel-MSFT) [inline docs/word/create-better-add-ins-for-word-with-office-open-xml.md:15]:
```suggestion
So what are your options when you need to add rich content to a document, such as images, formatted tables, charts, or even just formatted text? Start with the APIs available through the [WordApi requirement sets](/javascript/api/requirement-sets/word-api-requirement-sets) to see if they provide what you need. For an example, see the [Insert formatted text](https://github.com/OfficeDev/office-js-snippets/blob/prod/samples/word/25-paragraph/insert-formatted-text.yaml) code snippet in [Script lab](https://appsource.microsoft.com/product/office/wa104380862) on Word.
```

File: docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md [hunk 6]
```diff
@@ -28,7 +28,7 @@ Follow these steps to configure the add-in project to use a shared runtime.
 
 1. Start Visual Studio Code and open the add-in project you generated.
 1. Open the **manifest.xml** file.
-1. Replace (or add) the following `<Requirements>` section XML to require the [shared runtime requirement set](../reference/requirement-sets/shared-runtime-requirement-sets.md).
+1. Replace (or add) the following `<Requirements>` section XML to require the [shared runtime requirement set](/javascript/api/requirement-sets/shared-runtime-requirement-sets.md).
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md:31]:
```suggestion
1. Replace (or add) the following `<Requirements>` section XML to require the [shared runtime requirement set](/javascript/api/requirement-sets/shared-runtime-requirement-sets).
```

File: docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md [hunk 7]
```diff
@@ -75,7 +75,7 @@ Follow these steps to configure the add-in project to use a shared runtime.
     ```
     
     > [!NOTE]
-    > If your add-in includes the `Runtimes` element in the manifest (required for a shared runtime) and the conditions for using Microsoft Edge with WebView2 (Chromium-based) are met, it uses that WebView2 control. If the conditions are not met, then it uses Internet Explorer 11 regardless of the Windows or Microsoft 365 version. For more information, see [Runtimes](../reference/manifest/runtimes.md) and [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
+    > If your add-in includes the `Runtimes` element in the manifest (required for a shared runtime) and the conditions for using Microsoft Edge with WebView2 (Chromium-based) are met, it uses that WebView2 control. If the conditions are not met, then it uses Internet Explorer 11 regardless of the Windows or Microsoft 365 version. For more information, see [Runtimes](/javascript/api/manifest/runtimes.md) and [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md:78]:
```suggestion
    > If your add-in includes the `Runtimes` element in the manifest (required for a shared runtime) and the conditions for using Microsoft Edge with WebView2 (Chromium-based) are met, it uses that WebView2 control. If the conditions are not met, then it uses Internet Explorer 11 regardless of the Windows or Microsoft 365 version. For more information, see [Runtimes](/javascript/api/manifest/runtimes) and [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
```

**Comment by AlexJerabek** [comment]
Thanks @samantharamon and @ElizabethSamuel-MSFT. The concerns with the .md extensions in links and the redirection links should be addressed.

BTW, @ElizabethSamuel-MSFT, is there a way to add wildcards to redirects?

**Comment by ElizabethSamuel-MSFT** [comment]
@AlexJerabek Not sure about adding wildcards. However, the [contributor guide](https://review.docs.microsoft.com/en-us/help/contribute/retire-rename-many-articles?branch=main#how-can-you-make-the-task-simpler-can-you-use-native-git-functionality-to-minimize-the-apparent-change) mentions that if you have a large number of files, you can create a ticket in Sitehelp to request Akamai-level redirections. Might be worth asking about.

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**